src/lcm/LifeCycleStates.cc

```diff
@@ -2089,7 +2235,8 @@ void LifeCycleManager::trigger_disk_lock_success(int vid)
         {
             if (auto image = ipool->get(id))
             {
-                switch (image->get_state()) {
+                switch (image->get_state())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The switch statement here is missing a default statement. The default statement will catch any case values that are not explicitly handled by a case statement.  This prevents bugs if a case value is accidentally forgotten or new enum values are added later